index.js

```diff
@@ -21,18 +21,20 @@ function createClient() {
 
 const client = createClient();
 
-app.get("/", (request, res) => {
-    res.redirect(client.getProvider().getRedirect(redirectUri, scopes));
+app.get('/', (request, reply) => {
+    // Redirect clients who visit `/` to the authorization page.
+    reply.redirect(client.getProvider().getRedirect(redirectUri, scopes));
 });
 
-app.get("/callback", (request, reply) => {
+app.get('/callback', (request, reply) => {
+    // Respond with either an error, or with the information the client can use to authorize with.
     const oauth = client.getProvider();
 
     oauth.attempt(redirectUri, request.query)
         .then(res => reply.json(oauth.getTokens()))
-        .catch(err => reply.json(err));
+        .catch(reply.json);
```

Comment:
Assuming `reply.json` is bound rather than a method or is a closure itself, I'll check really quick.